TechnicalDocuments/FastlaneMatch.md

```diff
@@ -0,0 +1,147 @@
+Fastlane Match
+==============
+
+Fastlane match is a tool for managing certificates and provisioning profiles centrally. It downloads certificates and profiles from a GitHub repository or Google Cloud Storage bucket. For safety, centrally stored files are encrypted.
+
+## Installing Certificates and Provisioning Profiles Locally
+
+For day to day development work it usually suffices to navigate to the project directory in a terminal shell and run the command
+
+```shell
+ $ fastlane match development --readonly
+```
+
+Certificates and provisioning profiles for enterprise builds (Hockey App) or release builds (Testflight and the App Store) can also be downloaded. Normally, this is not needed but might be necessary if you want to run UI tests locally or work on updating our fastlane build scripts.
```

Comment:
I don't think this is really needed for running UI tests, but it is definitely needed for making manual builds.

```suggestion
Certificates and provisioning profiles for enterprise builds (Hockey App) or release builds (Testflight and the App Store) can also be downloaded. Normally, this is not needed but might be necessary if you want to make a release build locally or work on updating our fastlane build scripts.
```